ext/wscript

```diff
@@ -0,0 +1,14 @@
+def configure(conf):
```

Comment:
Perhaps dedupe these by adding `_recurse` function as seen in other files where the configure/build steps match?

Reply:
done